types/pdfkit/index.d.ts

```diff
@@ -333,10 +339,17 @@ declare namespace PDFKit.Mixins {
         expanded?: string;
         actual?: string;
     }
+    interface StructureElementOptions {
+        title?: string;
+        lang?: string;
+        alt?: string;
+        expanded?: string;
+        actual?: string;
+    }
     interface PageMarking {
         tag: string;
         structContent?: PDFStructureContent;
-        options?: MarkingOptions;
+        options?: StructureElementOptions;
```

Comment:
These options are passed to markStructureContent, so the statement above applies and the line must be reverted.